conf/modules.config

```diff
@@ -51,7 +51,7 @@ process {
 
     withName: PROKKA {
         publishDir = [
-            path: { "${params.outdir}/prokka" },
+            path: { "${params.outdir}/annotation/prokka" },
```

Comment:
Shouldn't this also have `meta.id`?

Reply:
The nf-core/prokka module assigns the output already in sample subfolders:
https://github.com/nf-core/modules/blob/master/modules/prokka/main.nf#L42

Hence I suggested to update other modules as well to not get confused. But I think it's fine to go like this with our couple of modules.

Reply:
Perfect OK :)